src/index.html

```diff
@@ -11,12 +11,105 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+
+      <nav class="navigation">
+        
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a
+              href="#apple"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a
+              href="#samsung"
+              class="nav-link"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a
+              href="#smartphones"
+              class="nav-link"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="navigation-item">
+            <a
+              href="#laptops-computers"
+              class="nav-link"
+              data-qa="hover"
```

Comment:
Don't use spaces in the `<a>` tag's `href` property. Anchor links start with the `#` symbol and should not contain spaces.